internal/pkg/runtime/platform/aws/aws.go

```diff
@@ -117,7 +117,7 @@ func IsEC2() (b bool) {
 
 // Name implements the platform.Platform interface.
 func (a *AWS) Name() string {
-	return "AWS"
+	return "aws"
```

Comment:
We've talked about making these consistent with the kernel args. I ran into issues with mismatches and figured now is as good a time as any to fix this.